docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -224,15 +224,17 @@ def tags_from_registry
         @tags_from_registry ||=
           begin
             client = docker_registry_client
-
-            client.tags(docker_repo_name, auto_paginate: true).fetch("tags")
+            repo = docker_repo_name
+            client.tags(repo, auto_paginate: true).fetch("tags")
```

Comment:
Is the above necessary?

Reply:
This looks good as well. I was eliminating to have two function calls to docker_repo_name (one here and another in the rescue block )so implemented repo = docker_repo_name as similar thing was done in [file_parser.rb](https://github.com/dependabot/dependabot-core/blob/main/docker/lib/dependabot/docker/file_parser.rb#L105)